frontend/src/components/Graph/index.tsx

```diff
@@ -189,10 +189,10 @@ function Graph({
 							unit: xAxisTimeUnit?.unitName || 'minute',
 							stepSize: xAxisTimeUnit?.stepSize || 1,
 							displayFormats: {
-								millisecond: 'HH:mm:ss',
-								second: 'HH:mm:ss',
-								minute: 'HH:mm',
-								hour: 'MM/dd HH:mm',
+								millisecond: 'hh:mm:ss aaa',
+								second: 'hh:mm:ss aaa',
+								minute: 'hh:mm aaa',
+								hour: 'MM/dd hh:mm aaaaa',
```

Comment:
using shorter notation for `hour` to prevent overlapping of texts
![Screenshot 2023-01-17 at 9 29 41 PM](https://user-images.githubusercontent.com/1501599/212956519-1c287ec7-ada2-4640-8ee4-2805588dceef.png)

Reply:
I would find it easier to read a 24-hour format both on the x-axis and on the chart. The am/pm doesn't look good to me.